### DISCUSSION TITLE
Select EVU/Service Provider

### DISCUSSION THREAD

**Discussion by ChillLP** [Traewelling/traewelling#2476] [Ideas / Feature Requests]
If possible, when creating Manual checkins a Option to set the Providing EVU, would be nice.
E.g. i Check into my Trains that i Drive, which are DB Cargo Trains.

**Comment by MrKrisKrisu** [comment]
As mentioned in the `Current limitation` section, this is already possible via the API, but currently not implemented in this Form.

![image](https://github.com/Traewelling/traewelling/assets/4103693/30eb18a4-8494-41fe-9f39-306a744c9048)

**Reply by ChillLP** [reply]
So, If Träwelldroid were to Support Manual checkins i could Check into DB Cargo Trains?

**Reply by MrKrisKrisu** [reply]
As `DB Cargo` is not a company listed in DB HAFAS (as I know) - currently no.
